php.env.example

```diff
@@ -1,16 +1,22 @@
-DB_PASSWORD=secret
-DB_DATABASE=forums
-DB_USERNAME=forums
+DB_HOST="online-db"
+DB_DATABASE="forums"
+DB_USERNAME="forums"
+DB_PASSWORD="secret"
 
 # Set to 1 to enable maintenance mode, 2 to make it immutable
 MAINTENANCE_MODE=0
 MAINTENANCE_TITLE="Maintenance Mode"
 MAINTENANCE_MESSAGE="Okay faithful users...we're attempting to restore an older backup of the database...news will be posted once we're back"
 
-SITE_NAME=forums
+SITE_NAME=Online
 SITE_LANGUAGE=english
 
 # These sites are expected to live at your.url/forum, please make sure your url has the correct structure
-SITE_URL=https://staging.wetfishonline.com/forum
+SITE_URL=https://wetfishonline.com/forum
 
-OUTGOING_EMAIL=noreply@yourdomain.com
+OUTGOING_EMAIL=noreply@wetfishonline.com
+
+# Online-specific bits
+FORUM_NAME=Wetfish Online
+FORUM_LANGUAGE=english
+FORUM_URL=https://wetfishonline.com/forum
```

Comment:
Where are these values being used? It doesn't look like they are making any impact on my local build. 

`SITE_NAME`, `SITE_LANGUAGE`, and `SITE_URL` are the actual names for these environment variables. It looks like they are defined here twice?

Reply:
Good question, looked over my notes and grep'ed through the repo and I can't seem to find a why.
If you're confident we can rm those lines then away they go.

Reply:
Good question, looking through my notes and grepping the repo I can't find a why. Looks to me they like they can be removed.

Reply:
Removed in [https://github.com/wetfish/online/pull/135/commits/8e61b53155ceb6e0fa3d1f65faeaeb71c084fbee](https://github.com/wetfish/online/pull/135/commits/8e61b53155ceb6e0fa3d1f65faeaeb71c084fbee)